tests/test_idle_usd.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from telliot_core.apps.core import TelliotCore
+from web3.datastructures import AttributeDict
+
+from telliot_feed_examples.feeds.idle_usd_feed import idle_usd_median_feed
+from telliot_feed_examples.reporters.tellorflex import PolygonReporter
+
+
+@pytest.mark.skip("Run locally, github action testing failure")
```

Comment:
We don't need an end-to-end test for submitting using each new datafeed, because this makes the CI workflow take super long, it unnecessarily drains our test tokens, and since we add unit tests for fetching new datapoints with each new feed, we don't need to test submitting the actual value, as there are already end-to-end tests for the  `PolygonReporter`.

Also, all these added reporting tests are being skipped anyway given: `@pytest.mark.skip`